forge/db/models/User.js

```diff
@@ -53,7 +53,13 @@ module.exports = {
                     user.name = user.username
                 }
             },
-            beforeUpdate: (user) => {
+            beforeUpdate: async (user) => {
+                if (user._previousDataValues.admin === true && user.admin === false) {
+                    const currentAdmins = await app.db.models.User.scope('admins').findAll()
```

Comment:
It would be better to use `.count()` rather than `findAll()` - let the database do the work to count, rather than transfer all of the data back to the app for us to found.